FinSearchUnified/Tests/Subscriber/SubscriberTestCase.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace FinSearchUnified\Tests\Subscriber;
+
+use Enlight_Controller_Request_RequestHttp;
+use Enlight_Controller_Response_ResponseHttp;
+use FinSearchUnified\Tests\TestCase;
+use ReflectionException;
+use ReflectionMethod;
+
+class SubscriberTestCase extends TestCase
+{
+    /**
+     * @param string $controller
+     * @param Enlight_Controller_Request_RequestHttp $request
+     * @param Enlight_Controller_Response_ResponseHttp|null $response
+     *
+     * @return mixed
+     * @throws ReflectionException
+     */
+    protected function getControllerInstance(
+        $controller,
+        Enlight_Controller_Request_RequestHttp $request,
+        Enlight_Controller_Response_ResponseHttp $response = null
+    ) {
+
+        if (is_null($response)) {
+            $response = new Enlight_Controller_Response_ResponseHttp();
+        }
+
+        $reflectionMethod = new ReflectionMethod($controller, 'Instance');
+        $subject = $reflectionMethod->invoke(null, $controller, [$request, $response]);
+
+        if (method_exists($subject, 'initController')) {
```

Comment:
Use `is_callable` instead of `method_exists`.

Reply:
@ihanli Using this is instead of method_exists is failing the tests

Reply:
Hmmm, interesting. Maybe `is_callable` considers the magic method `__call` while `method_exists` doesn't. Anyway, `method_exists` seems to work better here.